kube.mk

```
@@ -35,59 +11,55 @@ generate-go: bin/controller-gen bin/gofumpt # Generate Go code.
 	gofumpt -w -s ./pkg/apis
 
 .PHONY: generate-manifests
-generate-manifests: bin/controller-gen # Generate manifests e.g. CRD, RBAC etc.
+generate-manifests: generate-crds generate-rbacs generate-server-rbacs # Generate manifests e.g. CRD, RBAC etc.
```

want this to be a doc?
```suggestion
generate-manifests: generate-crds generate-rbacs generate-server-rbacs ## Generate manifests e.g. CRD, RBAC etc.
```

---

Hmm good catch this was from before I just tacked it onto `generate`.

---

Oh wait I misread the diff. No I don't think its necessary, `make generate` will do the right thing. If you'd like it to be documented I have no problem with it showing up in make help. Do *you* want this to be a doc?

---

Ha It doesn't matter to me. Since this isn't a top level target, I'm fine with keeping it not doc'd

---

Yeah thats been my thinking.